install.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+# Velib MCP Server - One-Line Installation Script
+# Installs and configures the Velib MCP server for Claude Code
+
+set -e
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+# Configuration
+REPO_URL="https://github.com/dominicburkart/velib-mcp.git"
+INSTALL_DIR="$HOME/velib-mcp"
+CONFIG_DIR="$HOME/.config/claude-code"
+MCP_CONFIG_FILE="$CONFIG_DIR/mcp_servers.json"
+SERVER_PORT=8080
```

Comment:
Can we pick a more exotic port to avoid the probability of collisions for users?